src/evaluate.cpp

```diff
@@ -114,6 +116,15 @@ namespace Eval {
         }
   }
 
+  void NNUE::export_embedded_net() {
+    if (gHasEmbeddedNet) {
```

Comment:
Net embedding is done during compilation so you can make ``gHasEmbeddedNet`` a ``constexpr bool`` and change this into a compile time if ``if constexpr (gHasEmbeddedNet)``